Assets/Scripts/PickupResizeBall.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum ChangeSize
+{
+    Increase,
+    Decrease
+}
+
+public class PickupResizeBall : Pickup
+{
+    public ChangeSize changeSize;
+    public int sizeMultiplier;
+
+    void Start()
+    {
+        
+    }
+    
+    private void OnTriggerEnter(Collider other)
```

Comment:
Instead of putting the resizing logic in OnTriggerEnter, put it in the override OnPickup() function.